lib/userstack/client.rb

```diff
@@ -8,18 +8,20 @@ module Userstack
   # A class which wraps calls to Userstack API
   class Client
     # @param access_key [String] Userstack Access key
+    # @param use_ssl [TrueClass, FalseClass] Use ssl or not
+    # @param legacy [TrueClass, FalseClass] Legacy response
```

Comment:
You can use `[Boolean]` instead of `[TrueClass, FalseClass]`.

See also https://www.rubydoc.info/gems/yard/file/docs/Tags.md